### PR TITLE
Update team.mdx

### PR DESCRIPTION
**Pull request by psmulovics**

### CODE REVIEW

File: docs/team.mdx [hunk 1]
```diff
@@ -6,11 +6,31 @@ title: Zenith Team
...
+He previously led the EMEA Innovation Program for Morgan Stanley, seeding their Spatial Computing & Quantum Technology programs and developing numerous proof-of-concept projects across a number of disciplines relevant to this special interest group proposal.
+
+Working across numerous industries since 2001, Keith has earned a varied skillset that enables him to adapt quickly to new technologies and disciplines. This experience covers the construction, logistics, finance and technology sectors. He currently holds an affiliate position with the Scottish International Space Advisory Council (SISAC) and is an active member of the Global Scot network.
+
+Keith earned his degree in Business from the University of Strathclyde, having previously studied Genetics & Biotechnology at the University of Glasgow.
 </Bio>
 
 <Bio name="Patrick Downing" organisation="Morgan Stanley" image="/img/people/patrick-downing.png">
+Patrick Downing is a Vice President of Identity and Access Management focusing on strategic, core infrastructure initiatives across Morgan Stanley. 
+
```
Comment (padraignix) [inline docs/team.mdx:20]: Erroneous spacing issue

Comment (psmulovics) [reply docs/team.mdx:20]: I feel like I am blind, where?

Comment (padraignix) [reply docs/team.mdx:20]: Seems there are a few of them between the three bios. On the preview page the paragraph/sentence spacing is different. Looks like:

image

Comment (psmulovics) [reply docs/team.mdx:20]: Ah, I see your point. And do you think I should merge the two paragraphs or the opposite, change it to a real new line?

Comment (padraignix) [reply docs/team.mdx:20]: For this one I think it's fine to have as a single paragraph. Looks like it's the first two lines of each bio that have the same issue however. So would need to give them the same treatment.

Comment (psmulovics) [reply docs/team.mdx:20]: I copied them off the deck, so it might have had some special character in it

File: docs/team.mdx [hunk 2]
```diff
@@ -6,29 +6,23 @@ title: Zenith Team
...
 
 Keith earned his degree in Business from the University of Strathclyde, having previously studied Genetics & Biotechnology at the University of Glasgow.
 </Bio>
 
 <Bio name="Patrick Downing" organisation="Morgan Stanley" image="/img/people/patrick-downing.png">
-Patrick Downing is a Vice President of Identity and Access Management focusing on strategic, core infrastructure initiatives across Morgan Stanley. 
-
-He has helped establish and now leads the Quantum Technology efforts at Morgan Stanley, focusing on near-term use case identification and implementation. 
+Patrick Downing is a Vice President of Identity and Access Management focusing on strategic, core infrastructure initiatives across Morgan Stanley. He has helped establish and now leads the Quantum Technology efforts at Morgan Stanley, focusing on near-term use case identification and implementation.
```
Comment (padraignix) [inline docs/team.mdx:17]: It just kicked the spacing issue down a paragraph. Think there is something more structural at foot.

image

Comment (psmulovics) [reply docs/team.mdx]: Strange, it's a normal {children} tag

**Review by padraignix — APPROVED** [review]
Looks good. Approved after confirming on preview site.

**Pull request merged**